package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "fontaweswimm",
   "main": "src/fonts/fontaweswimm/style.css",
-  "version": "1.1.0",
+  "version": "1.1.3",
```

Comment:
I noticed we skipped updating this up to 1.1.2

Reply:
Yes - I was lazy MF